### ISSUE TITLE
Softmax model not converging

### ISSUE THREAD

**Issue by krishnakanthnakka** [aamir-mustafa/pcl-adversarial-defense#6]
Hello,

I ran the code python softmax_training.py with default parameters, the validattion loss is struck at 2.30 with accuracy 10%.  Can you please check the code once.

[
<img width="418" alt="Screen Shot 2019-12-06 at 13 14 22" src="https://user-images.githubusercontent.com/39269210/70322441-63c88480-182a-11ea-8707-a349785e330a.png">
](url)

**Comment by krishnakanthnakka** [comment]
specifications, 

CUDA 10.1, PyTorch 1.12

**Comment by krishnakanthnakka** [comment]
I found the error. There's an issue with my Kubernetes machine.
 The num_workers has to be set to zero.
